backend/config/settings/development.py

```diff
@@ -3,14 +3,14 @@
 
 DEBUG = True
 
-# Used for Debug Toolbar to show
-INTERNAL_IPS = ["127.0.0.1"]
-
 # TODO: Setup the real dev DB here
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
+        "ENGINE": "django.db.backends.postgresql",

```

Comment:
These are the settings for our local DB.

Reply:
The dev DB should be set using the environment variables like we have in our current config.